rotkehlchen/tests/api/test_manually_tracked_balances.py

```diff
@@ -704,45 +713,81 @@ def test_delete_manually_tracked_balances_errors(rotkehlchen_api_server):
     )
     assert_error_response(
         response=response,
-        contained_in_msg='labels": ["Missing data for required field',
+        contained_in_msg='ids": ["Missing data for required field',
         status_code=HTTPStatus.BAD_REQUEST,
     )
 
-    # wrong type for labels
+    # wrong type for ids
     response = requests.delete(
         api_url_for(
             rotkehlchen_api_server,
             "manuallytrackedbalancesresource",
-        ), json={'labels': 1},
+        ), json={'ids': 'asdf'},
     )
     assert_error_response(
         response=response,
-        contained_in_msg='"labels": ["Not a valid list',
+        contained_in_msg='"ids": ["Not a valid list',
         status_code=HTTPStatus.BAD_REQUEST,
     )
 
-    # wrong type for label entries
+    # wrong type for ids entries
     response = requests.delete(
         api_url_for(
             rotkehlchen_api_server,
             "manuallytrackedbalancesresource",
-        ), json={'labels': ['My monero wallet', 55]},
+        ), json={'ids': [2, 'My monero wallet']},
     )
     assert_error_response(
         response=response,
-        contained_in_msg='"labels": {"1": ["Not a valid string."',
+        contained_in_msg='"ids": {"1": ["Not a valid integer."',
         status_code=HTTPStatus.BAD_REQUEST,
     )
 
-    # delete non-existing label
+    # delete non-existing id
     response = requests.delete(
         api_url_for(
             rotkehlchen_api_server,
             "manuallytrackedbalancesresource",
-        ), json={'labels': ['My monero wallet', 'nonexisting']},
+        ), json={'ids': [1, 23]},
     )
     assert_error_response(
         response=response,
-        contained_in_msg='Tried to remove 1 manually tracked balance labels that do not exist',
+        contained_in_msg='Tried to remove 1 manually tracked balance ids that do not exist',
         status_code=HTTPStatus.BAD_REQUEST,
     )
+
+
+def test_update_manual_balance_label(rotkehlchen_api_server):
+    _populate_tags(rotkehlchen_api_server)
+    balances = _populate_initial_balances(rotkehlchen_api_server)
+    balances.sort(key=itemgetter('id'))
+    balance_to_patch_1 = balances.pop()
+    balance_to_patch_1['label'] = 'NEW PATCHED LABEL 1'
+    balance_to_patch_2 = balances.pop()
+    balance_to_patch_2['label'] = 'NEW PATCHED LABEL 2'
+    expected_balances = balances + [balance_to_patch_2, balance_to_patch_1]
+    response = requests.patch(
+        api_url_for(
+            rotkehlchen_api_server,
+            "manuallytrackedbalancesresource",
```

Comment:
```suggestion
            'manuallytrackedbalancesresource',
```

Reply:
This is not resolved. I think something in your editor is changing single to double quotes.

Reply:
Resolved. This was my brain, not the editor :)